Core/GDCore/Extensions/Metadata/ParameterMetadata.h

```diff
@@ -156,21 +172,20 @@ class GD_CORE_API ParameterMetadata {
    * (or more, i.e: an object group).
    *
    * \see gd::ParameterMetadata::GetType
+   * \deprecated Use gd::ValueTypeMetadata instead.
    */
   static bool IsObject(const gd::String &parameterType) {
-    return parameterType == "object" || parameterType == "objectPtr" ||
-           parameterType == "objectList" ||
-           parameterType == "objectListOrEmptyIfJustDeclared" ||
-           parameterType == "objectListOrEmptyWithoutPicking";
+    return gd::ValueTypeMetadata::TypeIsObject(parameterType);
   }
```

Comment:
Doesn't the "Is" prefix only work for object methods? It feels strange in a static context because there is no object as subject.

Reply:
I think this convention can work for any function, even static ones, because it's still properly english ("is XXX something?" -> yes it is/no it's not -> it's definitely a boolean that you return) (while "XXX is something" -> is it an affirmation or a question?). 

At least that's what the folks choose for React (`shouldComponentUpdate` and not `componentShouldUpdate`).

Reply:
ok, I'll do the change